examples/fields/SmartDateField.README.md

```diff
@@ -0,0 +1,61 @@
+# SmartDateField
+
+## Why
+
+The `SmartDateField` lets you worry about operations on `Date`s as objects, without worrying about parsing dates, or formatting dates.  Currently the `SmartDateField` can parse 9 date types without any format strings.  This lets you concentrate on business logic, without worrying about incoming date formats.  This let's you confidently perform arithmetic and comparisons in `compute`, `recordCompute`, `batchRecordsCompute`, and `validate` on `Date`s.  `SmartDateField` has an `fString` argument that controls how the date is serialized after `validate` is finished.  Flatfile users no longer have to convert in and out of dates to maintain formatting.
+
+## What
+
+There are comprehensive tests for `SmartDateField`, the most demonstrative being [DateSheet.spec.ts](./DateSheet.spec.ts) .  This shows how dates can be compared as first class objects in a `recordCompute` function, and then formatted to a different persistence formate.
+
+
+### Assumptions made
+
+We will add options to override these assumption as needed.
+
+Date's without an Time of Day (TOD) and without a timezone (TZ)  will be interpreted as 00:00:00 in GMT.  By default JS dates are formatted to 00:00:00 in the local timezone, this leads to inconsistent results based on how the server is setup.  00:00:00 is also the the behavior of pandas timestamp
```

Comment:
"We chose `00:00:00` because that is the design of the pandas Timestamp"

I want the sentence because it shows users our thought process and that we have considered this.

Reply:
I was just wondering if our design decision to use X internally was relevant here. That's all